src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/quick-start-nrql-condition.mdx

```diff
@@ -0,0 +1,73 @@
+
+Your condition only needs two attributes: a query and a threshold. Your query determines what you want to be notified about, and the threshold determines when you’ll be notified. In our example, you want to be notified about a host’s CPU usage (query) when usage is above 70% (threshold).
+
+Follow these steps to write your first alerts condition using the NRQL query builder. Once you're done, you'll have a working alerts condition.
+
+## Prerequisites [#pre-nrql]
+
+If you haven't already, [create a New Relic account](https://newrelic.com/signup) before you start. It's free, forever. Then, send your data into New Relic to query. Policies also hold alerts conditions. You can either create a [policy](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/) before starting a query or while filling out the **Create Alert** form used later.
+
+## Step 1: Write your query [#step1-nrql]
+
+Determine what alerts condition you want to query before continuing. You can create an alerts condition from any NRQL query. Find the NRQL query builder either at the top or bottom of the New Relic One platform.
+
+![A screenshot showing where you can find the NRQL query builder in New Relic One.](./images/find-builder.png "A screenshot showing where you can find the NRQL query builder in New Relic One.")
+<figcaption>
+  The graph icon at the top includes the **Create alerts** button that will be used in this example. Click the **Open in Query builder** button after using the bottom NRQL query choice to reach the same page.
+</figcaption>
+
+In our example, our condition shows us each host’s CPU utilization. Write out the NRQL query as seen in the image below to track each host individually.
+
+![A screenshot of a NRQL query.](./images/nrql-query.png "A screenshot of a NRQL query.")
+<figcaption>
+  This query finds the average CPU utilization from a system and then breaks it down (facets) by individual host names.
+</figcaption>
+
+Our NRQL query language is a powerful tool that lets you write queries on any entity and its attributes. For more on NRQL, see [NRQL syntax](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/#syntax).
+
+Once you create a valid NRQL query, click on the “Create alert” button.
+
+![A screenshot of a NRQL query with an arrow pointing at the "Create alert" button.](./images/create-alert-button.png "A screenshot of a NRQL query with an arrow pointing at the "Create alert" button.")
+<figcaption>
+  Press enter to enable the "Create alert" button.
+</figcaption>
+
+## Step 2: See your query results [#step2-nrql]
+
+Visualize your NRQL alerts condition on New Relic One to make it easier for you to see how your systems are performing. In our example, our query shows how each host’s average CPU utilization is performing. A violation will open if any host’s average CPU utilization is over 70% and can alert us.
```

Comment:
I'm not sure what the first sentence is saying here.

I think it's more like... "On the NRQL condition creation page, name your condition and tweak your NRQL condition, if necessary."